### PR TITLE
docs(zh): update

### PR DESCRIPTION
**Pull request by huzhengen**
add note about push
expose START_LOCATION
addRoute, getRoutes

### CODE REVIEW

File: docs/zh/api/README.md [hunk 1]
```diff
@@ -321,6 +321,26 @@ sidebar: auto
 
   当前路由对应的[路由信息对象](#路由对象)。
 
+### router.START_LOCATION
+
+- 类型: `Route`
```
Comment (Jinjiang) [inline docs/zh/api/README.md]:
```suggestion
- 类型：`Route`
```

File: docs/zh/api/README.md [hunk 2]
```diff
@@ -321,6 +321,26 @@ sidebar: auto
...
+
+  初始路由地址，表示路由开始的[路由信息对象](#路由对象)。可用在导航守卫以区分初始导航。
```
Comment (Jinjiang) [inline docs/zh/api/README.md]:
```suggestion
  以[路由对象](#路由对象)的格式展示初始路由地址，即路由开始的地方。可用在导航守卫中以区分初始导航。
```

File: docs/zh/api/README.md [hunk 3]
```diff
@@ -400,6 +422,8 @@ const resolved: {
 
 ### router.addRoutes
 
+*废弃*: 使用 [`route.addRoute()`](#router-addroute) 代替。
```
Comment (Jinjiang) [inline docs/zh/api/README.md]:
```suggestion
*已废弃*：使用 [`route.addRoute()`](#router-addroute) 代替。
```

File: docs/zh/api/README.md [hunk 4]
```diff
@@ -408,6 +432,36 @@ router.addRoutes(routes: Array<RouteConfig>)
...
+
+函数签名:
+
+```ts
+addRoute(parentName: string, route: RouteConfig): () => void
+```
+
+### router.getRoutes
+
+获取所有激活的路由记录列表。**注意只有记录的属性才被视为公共 API**，避免使用任何其他属性，例如 `regex`，因为它在 Vue Router 4 中不存在。
```
Comment (Jinjiang) [inline docs/zh/api/README.md]:
```suggestion
获取所有活跃的路由记录列表。**注意只有文档中记录下来的 property 才被视为公共 API**，避免使用任何其它 property，例如 `regex`，因为它在 Vue Router 4 中不存在。
```

**Review by Jinjiang — CHANGES_REQUESTED** [review]

**Review by Jinjiang — APPROVED** [review]

**Comment by posva** [comment]
Thank you both!

**Pull request merged**